### PR TITLE
Add support for etcd_endpoints key

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: calico.py [hunk 1]
```diff
@@ -665,12 +665,16 @@ def main():
...
-                                        ETCD_AUTHORITY_DEFAULT)
-    os.environ[ETCD_AUTHORITY_ENV] = etcd_authority
-    _log.debug("Using ETCD_AUTHORITY=%s", etcd_authority)
+    etcd_authority = network_config.get(ETCD_AUTHORITY_KEY)
+    etcd_endpoints = network_config.get(ETCD_ENDPOINTS_KEY)
+    if etcd_authority:
+        os.environ[ETCD_AUTHORITY_ENV] = etcd_authority
+        _log.debug("Using %s=%s", ETCD_AUTHORITY_ENV, etcd_authority)
+    if etcd_endpoints:
+        os.environ[ETCD_ENDPOINTS_ENV] = etcd_endpoints
```
Comment (caseydavenport) [inline calico.py]: Presumably libcalico is validating these values?

File: calico.py [hunk 2]
```diff
@@ -665,12 +665,16 @@ def main():
     _log.debug("Loaded network config:\n%s",
                json.dumps(network_config, indent=2))
 
-    # Get the etcd authority from the config file. Set the
+    # Get the etcd configuration from the config file. Set the
     # environment variable.
```
Comment (caseydavenport) [inline calico.py]: variable**s**

**Comment by caseydavenport** [comment]
Failing tests, and coveralls is showing 63% coverage?

File: calico.py (hunk 1 above)
Comment (tomdee) [inline calico.py]: Yes

**Comment by tomdee** [comment]
OK, updates made. Just waiting for libcalico changes from @matthewdupre

**Comment by caseydavenport** [comment]
Circle tests are still failing due to an import error.

**Comment by caseydavenport** [comment]
I think it needs a test.  Should be quick, but coverage has fallen below 100%.

**Pull request merged**